_extensions/posit-docs/theme.scss

```diff
@@ -180,7 +150,7 @@ body {
   padding-top: 1px;
 }
 
-/* Posit logo - footer 
+/* Posit logo - footer
```

Comment:
pre-existing: should we remove this commented-out code?

Reply:
Since it is fairly difficult to track what styles do what - esp since we have light and dark theme files, I would rather keep the "comments" for what each class does

Reply:
@AshleyHenry15 - in this case, it's commenting out styles, as well. The `#footer-logo` is commented out.